scripts/oracle/config/hko.json

```diff
@@ -0,0 +1,334 @@
+{
+    "assets": [
```

Comment:
hard to synchronize this file

Reply:
can you move this file to: https://github.com/parallel-finance/apps-config under oracle directory, same change can be needed on oracle-client side